Doc/library/typing.rst

```diff
@@ -2651,6 +2652,35 @@ Functions and decorators
    runtime we intentionally don't check anything (we want this
    to be as fast as possible).
 
+.. decorator:: copy_kwargs(source_func)
+
+    Cast the decorated function's call signature to the *source_func*'s.
+
+    Use this decorator enhancing an upstream function while keeping it's
+    call signature.
+    Returns the original function with the *source_funcs* call signature.
+
+    Usage::
+
+        from typing import copy_kwargs, Any
+
+        def upstream_func(a: int, b: float, *, double: bool = False) -> float:
+            ...
+
+        @copy_kwargs(upstream_func)
+        def enhanced(
+            a: int, b: float, *args: Any, double: bool = False, **kwargs: Any
+        ) -> str:
+            ...
+
+    .. note::
+
+       Include ``*args`` and ``**kwargs`` in the signature of the decorated
+       function in order to avoid a :py:class:`TypeError` when the call signature of
+       *source_func* changes.
+
+   .. versionadded 3.14
```

Comment:
This is a comment, remember to include the colons. See:

https://devguide.python.org/documentation/markup/#paragraph-level-markup